Recipes/Along-slope-velocities.ipynb

```diff
@@ -23,6 +23,7 @@
    "outputs": [],
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.                      esm_datastore.search(variable='v',filename='ocean.nc'))
</pre><p>Is there something else (ideally, more 'physical') that we could search on besides filename? Filenames are going to be non-standard between each experiment, so it's best if we can show people how to search on "what they actually want" as opposed to "the magic we know works".</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/COSIMA/cosima-recipes/pull/458/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>